### ISSUE TITLE
Dependence on diaphragmatic breathing

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5900]
**Preferred term label:** Dependence on diaphragmatic breathing

**Synonyms**

**Definition (free text, please give PubMed ID)** Reduced ability for thoracic breathing leading to a dependence on diaphragmatic breathing.

**Parent term (use hpo.jax.org/app)** Intercostal muscle weakness HP:0004878; Functional respiratory abnormality HP:0002795

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@Aggriese I cannot find a good PubMed reference for this feature. Since diaphragmatic breathing is the norm, I am not sure if Dependence on diaphragmatic breathing is a good term name, and wonder if weakened thoracic breathing or something like that would be more accurate.

**Comment by mgriese** [comment]
The term is useful in the context of spinal cord injury. C5 to C6 spinal lesions have alterations of the intercostal and abdominal wall muscles, however the diaphragm (C3-C5) is intact. Thus patients can rely on their diaphragmatic function and do not need ventilatory support.

**Comment by pnrobinson** [comment]
@mgriese  I think this is basically a synonym of this
Intercostal muscle weakness HP:0004878
https://hpo.jax.org/app/browse/term/HP:0004878

**Comment by mgriese** [comment]
Diaphragmatic would (usually) not coded as intercostal; these are 2 different level of the respiratory driving force.

**Comment by pnrobinson** [comment]
I mean if you have (severe) Intercostal muscle weakness then you are going to be dependent on diaphragmatic beathing.

**Comment by mgriese** [comment]
Ok, I understand, thought: synonym...

**Comment by pnrobinson** [comment]
I will add this as a related synonym to  https://hpo.jax.org/app/browse/term/HP:0004878
